suite-native/atoms/src/AssetItem.tsx

```diff
@@ -16,6 +16,11 @@ type AssetItemProps = {
     iconName: CryptoIconName;
 };
 
+export const assetColorConfig: Record<string, string> = {
```

Comment:
I would go with simple `as const` and you can remove whole Record<string, string> thing.

Reply:
I tried this, but ran into error with index signature when passing the prop to progress bar (see type of `cryptoCurrencySymbol`)

Reply:
I would also have this config somewhere in another file but for the MVP it's too nitpick-y, I suppose :)

Reply:
I thought about it as well. Problem is that this is in `/atoms` and if I put it in `/module-home`, it will create circular dependency. What about `constants.ts` in src of atoms? @Nodonisko @dahaca

Reply:
When circular dependecy occurs it's strong signal that you need new package that will be used in both places. We will probably need some kind of config/constants package, but I think we should not do that in this PR. Let's only add TODO comment here, that this needs to be solved.

Reply:
Yep, will do.